PowerEditor/src/Notepad_plus.cpp

```diff
@@ -3947,7 +3947,8 @@ bool Notepad_plus::saveScintillaParams()
 
 bool Notepad_plus::addCurrentMacro()
 {
-	vector<MacroShortcut> & theMacros = (NppParameters::getInstance())->getMacroList();
+	NppParameters * pNppParam = NppParameters::getInstance();
+	vector<MacroShortcut> & theMacros = pNppParam->getMacroList();
 
 	int nbMacro = theMacros.size();
 

```

Comment:
Side note, ["drive by fix"](http://www.google.com/search?q=code+%22drive+by+fix%22) is a bit of [microspeak](http://blogs.msdn.com/b/oldnewthing/archive/tags/microspeak/).